docker/service/Dockerfile

```diff
@@ -2,4 +2,4 @@ FROM adoptopenjdk/openjdk11:slim
 
 COPY gp2gp.jar /
 
-ENTRYPOINT java -jar /gp2gp.jar --port 8080
+ENTRYPOINT java -jar /gp2gp.jar --port 9000
```

Comment:
Remove `--port 9000` this is not an option supported by java and doesn't do anything